### ISSUE TITLE
`pipel`-tab gives weird completions

### ISSUE THREAD

**Issue by dotsdl** [xonsh/xonsh#3577]
## xonfig

<details>

```
$ xonfig

+------------------+-----------------+
| xonsh            | 0.9.14          |
| Python           | 3.8.2           |
| PLY              | 3.11            |
| have readline    | True            |
| prompt toolkit   | 3.0.5           |
| shell type       | prompt_toolkit2 |
| pygments         | 2.6.1           |
| on posix         | True            |
| on linux         | True            |
| distro           | arch            |
| on darwin        | False           |
| on windows       | False           |
| on cygwin        | False           |
| on msys2         | False           |
| is superuser     | False           |
| default encoding | utf-8           |
| xonsh encoding   | utf-8           |
| encoding errors  | surrogateescape |
+------------------+-----------------+

```

</details>

## Expected Behavior

When typing `pipel` and hitting tab, I expect to see completions related to files or commands matching that initial text.

## Current Behavior

When typing `pipel` and hitting tab, this is what I currently see. This occurs as well when running under `xonsh --no-rc`:

![Screenshot from 2020-05-15 13-16-00](https://user-images.githubusercontent.com/1326811/82092502-44256a00-96ae-11ea-8d41-de245efedd70.png)

## Steps to Reproduce

Type `pipel` and hit tab.

**Comment by laloch** [comment]
Hi @dotsdl, thanks for reporting!
The issue, you are reporting, has most probably been already resolved by (not yet released) #3527. You can either switch to the current git master branch to test the change, or wait for v0.9.19.

**Comment by dotsdl** [comment]
Awesome, thanks @laloch! I'll watch out for it!
